ec/ecresource/deploymentresource/schema_kibana.go

```diff
@@ -93,3 +97,38 @@ func kibanaTopologySchema() *schema.Schema {
 		},
 	}
 }
+
+func kibanaConfig() *schema.Schema {
+	return &schema.Schema{
+		Type:             schema.TypeList,
+		Optional:         true,
+		MaxItems:         1,
+		DiffSuppressFunc: suppressMissingOptionalConfigurationBlock,
+		Description:      `Optionally define the Kibana configuration options for the Kibana Server`,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				// User settings
```

Comment:
I don't think this line is necessary? the fields already start with `"user_settings_*"`